contrib/charts/cert-manager/templates/NOTES.txt

```diff
@@ -1,15 +1,58 @@
-cert-manager has been deployed successfully!
+{{- if .Release.IsInstall }}
+cert-manager and it's CRDs have been installed successfully!
 
 In order to begin issuing certificates, you will need to set up a ClusterIssuer
-or Issuer resource (for example, by creating a 'letsencrypt-staging' issuer).
+or Issuer resource.
+
+You can install the default letsencrypt ClusterIssuers (staging and production)
+by upgrading this chart and specifying a letsencryptEmail value:
+
+helm upgrade {{ .Release.Name }} --reuse-values --set letsencryptEmail= stable/cert-manager
```

Comment:
I think we need to highlight "The Helm chart provides a convenience method to automatically create some default issuers for Letsencrypt staging and prod" - this reads like it's the only way to enable Letsencrypt, and I'm specifically trying to steer people away from this "configure cert-manager itself to get LE certs" mentality that kube-lego has instilled in users.

Given that Issuer resources are targeted at *administrators* of a cluster, I think it's important they understand what resources and where are doing what, in order to take consideration of security implications etc (i.e. proper RBAC configuration, or being aware of the security implication of ClusterIssuer vs Issuer, and how that effects 'ambient credentials')